platform/broadcom/sonic-platform-modules-cel/debian/platform-modules-dx010.init

```diff
@@ -33,45 +33,36 @@ case "$1" in
 start)
         echo -n "Setting up board... "
 
-        rmmod i2c-i801
-        rmmod i2c-ismt
         modprobe i2c-dev
-        modprobe i2c-i801
-        modprobe i2c-ismt
         modprobe i2c-mux-pca954x
         modprobe dx010_wdt
         modprobe leds-dx010
         modprobe lm75
 
-        found=0
-        for devnum in 0 1; do
-                devname=`cat /sys/bus/i2c/devices/i2c-${devnum}/name`
-                # iSMT adapter can be at either dffd0000 or dfff0000
-                if [[ $devname == 'SMBus iSMT adapter at '* ]]; then
-                        found=1
-                break
-                fi
-        done
+        # Detech 
```

Comment:
Typo: "Detect"?

Also please add more detail. This is not a useful comment.

Reply:
https://github.com/Azure/sonic-buildimage/pull/6889/commits/cb97d8914a63d3f38c7df0df982ecf0161b6a4ba